quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -120,11 +124,45 @@ impl Limits {
         unidirectional_data_window,
         u64
     );
-    setter!(
-        with_max_open_bidirectional_streams,
-        max_open_bidirectional_streams,
-        u64
-    );
+
+    /// Sets the max local and remote limits for bidi streams.
+    ///
+    /// The value `with_max_open_remote_bidirectional_streams` and
+    /// `with_max_open_local_bidirectional_streams` will be used instead
+    /// if set on the builder.
+    // TODO: Deprecate
+    pub fn with_max_open_bidirectional_streams(
+        mut self,
+        value: u64,
+    ) -> Result<Self, ValidationError> {
+        self.max_open_bidirectional_streams = value.try_into()?;
+        Ok(self)
+    }
+
+    /// Sets the max local limits for bidi streams
+    ///
+    /// The value set is used instead of `with_max_open_bidirectional_streams` when set.
```

Comment:
Do we need it to behave that way? In my mind it would be clear if `with_max_open_bidirectional_streams` was just a "helper" to set both local and remote values at the same time to the same value, so calling either of the local or remote specific methods afterwards would overwrite the individual value. I'm not sure we need the ability to still use the individual local or remote value if `with_max_open_bidirectional_streams` is called later.